communicate-on-pull-request-merged/action.yml

```diff
@@ -8,9 +8,12 @@ inputs:
   pr-comment:
     description: 'A comment to post on a pull request when code changes are merged'
     required: true
-  pr-label:
+  pr-label-to-add:
     description: 'The label to apply when a pull request is merged'
     default: 'status: included-in-next-release'
+  pr-label-to-remove:
+      description: 'The label to remove when a pull request is merged'
```

Comment:
Is this used for any matching, or just for information here?

Reply:
👋 It is just an information. Basically we are providing the possibility to remove the label whenever the pull request is merged. On a current _fastlane_ bot implementation, we remove  `status: needs-attention label `.